src/components/MewOverlay/MewOverlay.vue

```diff
@@ -246,17 +280,27 @@ export default {
     color: rgba(11, 40, 64, 0.72);
   }
   .mobile-title {
-    box-shadow: 0px 1px 6px rgba(11, 40, 64, 0.06), 0px 2px 8px rgba(11, 40, 64, 0.04);
+    box-shadow: 0px 1px 6px rgba(11, 40, 64, 0.06),
+      0px 2px 8px rgba(11, 40, 64, 0.04);
     position: fixed;
     top: 0;
     width: 100%;
+    height: 70px;
     z-index: 9999;
+
+    div {
+      line-height: 25px;
+    }
   }
-  .mobile-content {
-    margin-top: 68px;
-  }
+
   .mobile-btn {
     z-index: 99999;
   }
 }
-</style>
\ No newline at end of file
+</style>
+
+<style lang="scss">
```

Comment:
please add this in the same style script, don't create another one

Reply:
This is a global style.